ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -1624,7 +1604,90 @@ def overlay_routing_protocol_address_family(self):
         return overlay_routing_protocol_address_family
 
     @cached_property
-    def overlay_routing_protocol_peering_address(self):
+    def bgp(self):
+        return (
```

Comment:
please add a docstring to explain the meaning/purpose of the new facts.
```suggestion
        """
        Boolean telling if BGP Routing should be configured
        """
        return (
```

Reply:
Fixed.